### DISCUSSION TITLE
Wrapping Text in Entry

### DISCUSSION THREAD

**Discussion by EmberLynn** [dotnet/maui#9503] [Q&A] [answered]
As per the title, I am trying to have text wrap on an Entry. Here is the code for my Entry:

```XAML
<VerticalStackLayout IsVisible="{Binding Request.RequestConfig.IsDescriptionVisible}">
    <Label Text="{Binding Request.RequestConfig.DescriptionLabel}"/>
    <Entry Placeholder="Enter your sentence here" 
           Text="{Binding Request.Description}"
           VerticalTextAlignment="Start"
           HeightRequest="70"/>
</VerticalStackLayout>
```

And here is what my Entry looks like on the screen:
![Entry height and wrap](https://user-images.githubusercontent.com/46328589/185232069-a11b2e9f-363f-4520-8f5a-ae7e43fdb31f.png)

The beginning of the sentence in the picture is truncated because the text keeps going in one line. Is there a way to force the Entry to wrap so it appears as a paragraph on screen?

For reference, this is what it looks like in the Android 12 emulator. I am unsure if this performs differently on iOS.

**Comment by hartez** [comment, marked as answer]
The `Editor` control allows multi-line text with wrapping.

**Reply by EmberLynn** [reply]
Yes, thank you. I did discover this a few days after I posted!

**Reply by Isaac-Saleh** [reply]
Any further hints to what you discovered?? I have now googled editor text wrapping with .net maui and have had no luck. I am having the same issue of creating and input box that is larger than a single line of text but the text just continues off the screen. Any further help to point me in the right direction would be super appreciated. Ta

**Reply by jonmdev** [reply]
Use Editor for multitext not Entry.

**Reply by Isaac-Saleh** [reply]
I'll have a look into it. Great thanks
